fe1-web/src/features/wallet/store/__tests__/DigitalCashStore.test.tsx

```diff
@@ -0,0 +1,69 @@
+import 'jest-extended';
+
+import {
+  mockCBHash,
+  mockCoinbaseTransactionJSON,
+  mockKeyPair,
+  mockKPHash,
+  mockLaoId,
+  mockTransactionValue,
+} from '__tests__/utils';
+import { Transaction } from 'features/wallet/objects/transaction';
+import { getDigitalCashState } from 'features/wallet/reducer';
+
+import { DigitalCashStore } from '../DigitalCashStore';
+
+const mockTransaction = Transaction.fromJSON(mockCoinbaseTransactionJSON, mockCBHash).toState();
+
+const filledState = {
+  byLaoId: {
+    [mockLaoId.valueOf()]: {
+      balances: {
+        [mockKPHash.valueOf()]: mockTransactionValue,
+      },
+      allTransactionsHash: [mockTransaction.transactionId],
+      transactionsByHash: {
+        [mockTransaction.transactionId!]: mockTransaction,
+      },
+      transactionsByPubHash: {
+        [mockKPHash.valueOf()]: [mockTransaction.transactionId],
+      },
+    },
+  },
+};
+
+jest.mock('features/wallet/reducer/DigitalCashReducer');
+
+beforeAll(() => {
+  (getDigitalCashState as jest.Mock).mockReturnValue(filledState);
+});
+
+describe('Digital Cash Store', () => {
```

Comment:
It could be nice to add spaces between the `it`s.

Reply:
Noted !